gen/gen.go

```diff
@@ -405,68 +405,23 @@ var packageTemplate = `// Package {{.PackageName}} GENERATED BY THE COMMAND ABOV
 // {{ .Timestamp }}{{ end }}
 package {{.PackageName}}
 
-import (
-	"bytes"
-	"encoding/json"
-	"strings"
-	"text/template"
+import "github.com/swaggo/swag"
 
-	"github.com/swaggo/swag"
-)
-
-var doc = ` + "`{{ printDoc .Doc}}`" + `
+const docTemplate_{{ .InstanceName }} = ` + "`{{ printDoc .Doc}}`" + `
 
-type swaggerInfo struct {
-	Version     string
-	Host        string
-	BasePath    string
-	Schemes     []string
-	Title       string
-	Description string
-}
-
-// SwaggerInfo holds exported Swagger Info so clients can modify it
-var SwaggerInfo = swaggerInfo{
+// SwaggerInfo_{{ .InstanceName }} holds exported Swagger Info so clients can modify it
+var SwaggerInfo_{{ .InstanceName }} = &swag.SwaggerInfo{
 	Version:     {{ printf "%q" .Version}},
```

Comment:
Please rename the struct to Doc so the definition will not contain the swag string to times.
```
var SwaggerInfo_{{ .InstanceName }} = &swag.Spec {
```

Reply:
Done